pkgs/development/python-modules/github-webhook/default.nix

```diff
@@ -1,28 +1,25 @@
-{ stdenv
-, buildPythonPackage
-, fetchgit
+{ lib, buildPythonPackage, fetchPypi
 , flask
+, six
 }:
 
 buildPythonPackage rec {
   pname = "github-webhook";
-  version = "unstable-2016-03-11";
+  version = "1.0.2";
 
-  # There is a PyPI package but an older one.
-  src = fetchgit {
-    url = "https://github.com/bloomberg/python-github-webhook.git";
-    rev = "ca1855479ee59c4373da5425dbdce08567605d49";
-    sha256 = "0mqwig9281iyzbphp1d21a4pqdrf98vs9k8lqpqx6spzgqaczx5f";
+  src = fetchPypi {
```

Comment:
Are we certain this is the same code?

The homepage for this package on pypi is
* https://github.com/looking-for-a-job/github-webhooks.py

Before we were fetching from
* https://github.com/bloomberg/python-github-webhook